src/renderer/services/file-management-system/index.ts

```diff
@@ -78,6 +77,7 @@ export default class FileManagementSystem {
     user: string,
     serviceFields: Partial<UploadServiceFields> = {}
   ): Promise<UploadJob> {
+    console.log(metadata.file.originalPath); // TODO remove after pilot testing
```

Comment:
Is this TODO and similar for returning to after merge?

Reply:
Before - after testing a snapshot version on windows.  just wanna make sure the path look the way i expect, and implement some unit tests.